annotations/views/rest_views.py

```diff
@@ -292,7 +294,27 @@ def get_queryset(self, *args, **kwargs):
             queryset = queryset.filter(project_id=project_id)
         if position_type and position_type in DocumentPosition.TYPES:
             queryset = queryset.filter(position__position_type=position_type)
-        return queryset.order_by('-created')
+
+        quesryset = queryset.order_by('-created')
```

Comment:
queryset is misspelled; might create issues down the line (you return queryset)

Reply:
fixed